apps/site/pages/en/learn/test-runner/mocking.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Mocking in tests
+layout: learn
+authors: JakobJingleheimer
+---
+
+# Mocking in tests
+
+Tests should be deterministic: runnable in any order, any number of times, and always produce the same result. Proper setup and mocking make this possible.
```

Comment:
nit: would it make more sense to start with the paragraph that defines mocking, then add this one which explains why mocking?

Reply:
Actually yeah that does make sense. Will update over lunch